posix.mak

```diff
@@ -383,10 +383,13 @@ docs-prerelease: dmd-prerelease druntime-prerelease phobos-prerelease apidocs-pr
 
 docs : docs-latest docs-prerelease
 
-html : $(ALL_FILES) makebook
+html : html_files makebook
+
+html_files : $(ALL_FILES)
+
+makebook : book/build.d | $(STABLE_DMD)
+	cd book && PATH="$(STABLE_DMD_BIN_ROOT):$$PATH" rdmd build.d 6
```

Comment:
I'm not too familiar with GNU Makefile, but why subshell is no longer needed? does every command run in a subshell by default?

Reply:
With a bit of research, I found out that this is indeed redundant